centreon/tests/e2e/cypress/e2e/Resources-status/02-acknowledgments/index.ts

```diff
@@ -0,0 +1,649 @@
+import { Given, Then, When } from '@badeball/cypress-cucumber-preprocessor';
+
+import {
+  checkThatFixtureHostsExistInDatabase,
+  checkThatFixtureServicesExistInDatabase
+} from '../../../commons';
+import {
+  actionBackgroundColors,
+  checkIfNotificationsAreNotBeingSent,
+  checkIfUserNotificationsAreEnabled,
+  clearCentengineLogs,
+  hostChildInAcknowledgementName,
+  hostInAcknowledgementName,
+  insertAckResourceFixtures,
+  searchInput,
+  serviceInAcknowledgementName,
+  submitCustomResultsViaClapi,
+  tearDownAckResource,
+  tearDownResource,
+  typeToSearchInput
+} from '../common';
+
+before(() => {
+  cy.startWebContainer();
+});
+
+beforeEach(() => {
+  cy.intercept({
+    method: 'POST',
+    url: '/centreon/api/latest/monitoring/resources/acknowledge'
+  }).as('postAcknowledgments');
+  cy.intercept({
+    method: 'GET',
+    url: '/centreon/api/internal.php?object=centreon_topology&action=navigationList'
+  }).as('getNavigationList');
+  cy.intercept({
+    method: 'GET',
+    url: '/centreon/include/common/userTimezone.php'
+  }).as('getTimeZone');
+});
+
+Given('the user have the necessary rights to page Ressource Status', () => {
+  cy.loginByTypeOfUser({
+    jsonName: 'admin',
+    loginViaApi: true
+  });
+
+  cy.get(searchInput).should('exist');
+});
+
+Given(
+  'the user have the necessary rights to acknowledge & disacknowledge',
+  () => {
+    cy.getByLabel({ label: 'Acknowledge' }).should('be.visible');
+  }
+);
+
+Given(
+  'there are at least two resources of each type with a problem and notifications enabled for the user',
+  () => {
+    insertAckResourceFixtures();
+
+    checkIfUserNotificationsAreEnabled();
+
+    cy.refreshListing();
+  }
+);
+
+Given(
+  'a single resource selected on Resources Status with the "Resource Problems" filter enabled',
+  () => {
+    cy.contains('Unhandled alerts');
+
+    cy.contains(serviceInAcknowledgementName)
+      .parent()
+      .parent()
+      .find('input[type="checkbox"]:first')
+      .click();
+  }
+);
+
+Given('acknowledgment column is enabled in Resource Status', () => {
+  cy.get('[aria-label="Add columns"]').click();
+
+  cy.contains('State').click();
+
+  cy.get('[aria-label="Add columns"]').click();
+});
+
+When('the user uses one of the "Acknowledge" actions', () => {
+  cy.getByLabel({ label: 'Acknowledge' }).last().click();
+});
+
+When(
+  'the user fills in the required fields in the form with default parameters "sticky & persistent checked"',
+  () => {
+    cy.get('textarea').should('be.visible');
+
+    cy.getByLabel({ label: 'Notify' }).should('not.be.checked');
+
+    cy.getByLabel({ label: 'Sticky' }).should('be.checked');
+
+    cy.getByLabel({ label: 'Persistent' }).should('be.checked');
```

Comment:
```suggestion
```
The persistent checkbox will not be displayed very soon

Reply:
should i remove this too ? since we are removing the "persistant" stuff